lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -539,6 +587,7 @@ def build_entity(self):
                 self.param('serial_policy') is not None or
                 self.param('serial_policy_value') is not None
             ) else None,
+            disk_attachments=disks if disks else None,
```

Comment:
nothing should happen it put the disk on the first storage storage domain on the list ... in my case it always choose nfs_0 ....